server/internal/lsp/search/search_completion_list.go

```diff
@@ -222,6 +237,9 @@ func (s *Search) BuildCompletionList(
 					items = append(items, protocol.CompletionItem{
 						Label: member.GetName(),
 						Kind:  &member.Kind,
+
+						// At this moment, struct members cannot receive documentation
+						Documentation: nil,
```

Comment:
Is this because of how you decide which doc comment you assign to a symbol? (Las doc block parsed belongs to next identified symbol)

Reply:
In part, yes: I'd have to replicate my code to detect and assign doc comments in the struct parsing function as well in order to accept doc comments inside the struct, i.e. to accept something like

```c
struct Name {
  <* doc *>
  int a;
}
```

But also, I didn't do that because it doesn't compile (through `c3c`), even though the tree-sitter grammar accepts it.

We could, of course, just copy and paste the struct's own docs to all of its members, but that doesn't feel ideal, at least not at the moment.